### ISSUE TITLE
GraphQL: devnet fetching package function fails with internal server error

### ISSUE THREAD

**Issue by FrankC01** [MystenLabs/sui#19117]
This returns expected results:
```graphql
query Func2 {
object(
    address: "0x3"
  ) {
    asMovePackage {
      module(name: "sui_system") {
        function(name: "request_withdraw_stake") {
          ... TheFunction
        
        }
      }
    }
  }
}
fragment TheFunction on MoveFunction {
  function_name: name
  isEntry
  visibility
  typeParameters {
    constraints
  }
  returns: return {
    signature
  }
  parameters {
    signature
  }
}
```

This returns internal server error
```graphql
query Func {
object(
    address: "0x6681d8833d65a55f4b74de43f0f7f92f68eb1f96dbbc8eee4024bfa19ebf03ad"
  ) {
    asMovePackage {
      module(name: "parms") {
        function(name: "check_uints") {
          ... TheFunction
        
        }
      }
    }
  }
}
fragment TheFunction on MoveFunction {
  function_name: name
  isEntry
  visibility
  typeParameters {
    constraints
  }
  returns: return {
    signature
  }
  parameters {
    signature
  }
}
```

```json
{
  "data": {
    "object": {
      "asMovePackage": null
    }
  },
  "errors": [
    {
      "message": "Internal error occurred while processing request: Error reading package: VMError with status UNKNOWN_VERSION at location UNDEFINED",
      "locations": [
        {
          "line": 6,
          "column": 7
        }
      ],
      "path": [
        "object",
        "asMovePackage",
        "module"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
```

**Comment by amnn** [comment]
Thanks for the report @FrankC01

**Comment by FrankC01** [comment]
Changing just to the original `devnet` URL this appears to be fixed

**Comment by amnn** [comment]
Yes, this issue is specific to trying to use the legacy GraphQL with packages from a newer version of the network. Once we change our versioning and release model (9th September, v1.33), this problem will be resolved.

**Comment by amnn** [comment]
This has now been addressed as part of the 1.33 release, thanks again for the report Frank!
